client/src/pages/AdminSettingsPage.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import Sidebar from "../components/Sidebar";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faBook,
+  faBuildingColumns,
+  faBuildingUser,
+  faUsers,
+} from "@fortawesome/free-solid-svg-icons";
+
+const AdminSettingsPage = () => {
+  return (
+    <div className="flex w-full">
+      <Sidebar />
+      <div className="w-full">
+        <h1 className="font-bold p-3 text-lg border-b-2">Admin Settings</h1>
```

Comment:
Admin Settings header should stick at top when user scrolls down.
Refer to AdminAnnnouncement page header on how it was implemented.